.travis.yml

```diff
@@ -12,10 +12,9 @@ env:
     - PRESET=ARM_REL
     - PRESET=ARM_REL_MIN
 
-    - PRESET=STM32F4
+    - PRESET=STM32FX
```

Comment:
I believe this is more generic than specific. E.g. the LwIP solution is not F family specific. Neither the memory, rng.